src/lib/fetchers/clientFetcher.ts

```diff
@@ -11,7 +10,7 @@ type FetcherParameters = {
 };
 
 // POST, PUT, DELETE fetcher
-export function clientFetcher(params: FetcherParameters) {
+export function clientFetcher<DataType>(params: FetcherParameters): Promise<DataType | undefined> {
```

Comment:
Shouldn't we get rid of the undefined like we talked?

Reply:
Done on new revision